Yona/Yona/Challenges/TimeFrameTimeZoneChallengeViewController.swift

```diff
@@ -242,6 +246,8 @@ extension TimeFrameTimeZoneChallengeViewController {
                 self.setChallengeButton.alpha = 0.5
             }
             self.tableView.endUpdates()
+            
+            NSTimer.scheduledTimerWithTimeInterval(0.1, target: self, selector: Selector.tableReload, userInfo: nil, repeats: false)
```

Comment:
We need to investigate better regarding the problem with the deletion of rows that not update the UI, because this solution to call with a time the `tableView.reloadData` is wrong. Probably we have a issue in the the deletion logic.